nivacloud_logging/requests_trace.py

```diff
@@ -22,4 +22,8 @@ def add_headers(self, request, **kwargs):
         incoming_trace_id = LogContext.getcontext("trace_id")
         if incoming_trace_id:
             request.headers['Trace-Id'] = incoming_trace_id
+
+        incoming_user_id = LogContext.getcontext("user_id")
+        if incoming_user_id:
+            request.headers["User-Id"] = incoming_user_id
```

Comment:
question, this clearly is a case for 
![walrus](https://user-images.githubusercontent.com/736057/72416848-e0c71200-3777-11ea-9745-c2ee3bc4c59f.jpg)

should we request python 3.8 for nivacloud-logging? if yes, how do I express that in setup.py?

Reply:
Clearly. :joy: 

I'm not actually sure how you define a hard constraint on 3.8. I would just remove the `'Programming Language :: Python :: 3.7'`, etc. Not sure if the `'Programming Language :: Python :: 3'` stays or not...